src/extensions/default/HealthData/HealthDataManager.js

```diff
@@ -110,8 +110,8 @@ define(function (require, exports, module) {
     }
 
     /*
-     * Check if the health data is to be sent to the server. If the user has enabled tracking, health data will be sent once every 24 hours.
-     * Send health data to the server if the period is more than 24 hours.
+     * Check if the Health Data is to be sent to the server. If the user has enabled tracking, Health Data will be sent once every 24 hours.
+     * Send Health Data to the server if the period is more than 24 hours.
```

Comment:
Ah, I thought that was the official name, and I wanted to harmonize all the different notations a little.